examples/api/src/index.ts

```diff
@@ -43,6 +44,10 @@ class ExampleURLAdapter implements URLAdapter {
     return `${this.websiteURL}/a/${article.id}/${article.slug}`
   }
 
+  getPeeredArticleURL(peer: Peer, article: PublicArticle): string {
+    return `${this.websiteURL}/p/${peer.id}/${article.id}`
```

Comment:
but what if we implemented the pages as well, how would we differ between article and page urls

Reply:
@bishoymelek pages dont have a `/a/` or `/p/` prefix in the URL path. They just use the slug after the domain.